.github/workflows/docs.yml

```diff
@@ -24,6 +24,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v3
+      with:
+        repository: "main"
```

Comment:
Hm, I don't think this is right. I believe it'll look for a repository called "main" not a branch; looks like if you want a branch you need to use `ref:`.

But reading through the github workflow docs a bit, I think we're keying off the wrong event. Seems like we should be using `pull_request` rather than `pull_request_target`, see: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-workflow-when-a-pull-request-merges.